content/en/integrations/guide/azure-architecture-and-configuration.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Azure Architecture and Configuration
+kind: guide
+description: "Guide to architecture of the Datadog Azure integration and alternate configuration options"
+further_reading:
+- link: "https://docs.datadoghq.com/integrations/azure/"
+  tag: "Documentation"
+  text: "Azure Integration"
+---
+
+## Overview
+
+This guide provides detailed information and reference architectures for users configuring Datadog's Azure integration, as well as alternative configuration options for specific use cases.
+
+### Reference architectures
+
+The diagrams in this guide provide a visual representation of the configuration process and outcome when following the steps in the [Azure integration page][1]. This guide provides a detailed overview of Datadog's interaction with your Azure environment and answers common security, compliance, and governance questions.
+
+### Alternate configurations
+
+The setup processes documented in the [Azure integration page][1] are the recommended steps and result in the ideal configuration for the majority of users. Alternate configuration options in this document may be preferable for certain use cases. Any trade-offs in performance, features, or ease-of-management are outlined as needed.
+
+## Azure metric and data collection
+
+Enabling Datadog's Azure integration allows Datadog to:
+
+  - Discover and monitor all resources in all subscriptions within the given scope
+  - Automatically update the metric definitions, to ensure that all of the metrics available from Azure Monitor are collected
```

Comment:
Remove "the" in front of metric definitions, or maybe rewrite so it's "Automatically update discovered metric definitions"